task01.rb

```diff
@@ -2,11 +2,11 @@
 
 def task01(str)
   str.gsub!(/ /, "")
-  if str.scan(/[<>]=?\d+/)[0].length == str.length 
+  if str.scan(/[<>]?=?\d+/)[0].length == str.length 
```

Comment:
@MaraEfimova I believe this is going to return `true` in case of `> =9`, which is in fact invalid input...